core/jazz_logs/swagger/swagger.json

```diff
@@ -479,8 +479,12 @@
             "apigee_proxy"
```

Comment:
This list is probably incorrect. We only support logs for `lambda` & `apigateway`.

Reply:
done